benchmarks/multimodal/http/runner.py

```
@@ -0,0 +1,57 @@
+
+
+def gen_urls(seeds: list[str], n: int) -> list[str]:
+    return [f"{seeds[i % len(seeds)]}?cb={uuid.uuid4().hex}" for i in range(n)]
+
+
+async def _timed_fetch(url: str, timeout: float) -> tuple[float, str]:
+    start = time.perf_counter()
+    try:
+        await fetch_bytes(url, timeout=timeout)
+        return (time.perf_counter() - start, "success")
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)